src/components/PageStepper/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { OptionsContainer, Option } from './styles';
+
+interface PageStepperProps {
+  steps: Array<string>;
+  activeStep: number;
+  onStepClick: (newActiveStep: number) => void;
+}
+
+const PageStepper: React.FC<PageStepperProps> = ({
+  steps,
+  activeStep,
+  onStepClick,
+}) => {
+  const optionItem = (index: number, isActive: boolean, option: string) => {
+    return (
+      <Option isActive={isActive} key={option}>
+        <a onClick={() => onStepClick(index)}>{option}</a>
+      </Option>
+    );
+  };
```

Comment:
Se não for colocar nenhuma variável, desestruturação ou "cálculo" antes do _return_, pode retornar direto, usando `()` no lugar de `{}`.

```suggestion
  const optionItem = (index: number, isActive: boolean, option: string) => (
      <Option isActive={isActive} key={option}>
        <a onClick={() => onStepClick(index)}>{option}</a>
      </Option>
  );
```